frontend/src/entities/configuration/allowed-instance-types-configuration.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License").
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import {
+    Container,
+    ExpandableSection,
+    Header
+} from '@cloudscape-design/components';
+import React, { useMemo } from 'react';
+import { InstanceTypeMultiSelector } from '../../shared/metadata/instance-type-dropdown';
+
+export type AllowedInstanceTypesConfigurationProps = {
+    setFields: (SetFieldsFunction) => void;
```

Comment:
Not sure this PR is ready for review yet but pretty sure this should be `setFields: SetFieldsFunction`. I probably wasn't clear in my previous comment that `SetFieldsFunction` is the whole method signature instead of just the parameter signature.